backend/ops_api/ops/utils/auth_views.py

```diff
@@ -1,43 +1,44 @@
+import logging
+import sys
 import traceback
 from typing import Union
 
 import requests
-from flask import Response, jsonify, request
+from flask import Response, current_app, jsonify, request
 from flask_jwt_extended import create_access_token, create_refresh_token, get_jwt_identity, jwt_required
-from ops_api.ops.utils.auth import get_jwt, oauth
+from ops_api.ops.utils.auth import create_oauth_jwt, oauth
 from ops_api.ops.utils.user import process_user
 
+logging.basicConfig(stream=sys.stderr, level=logging.DEBUG)
```

Comment:
instead of independently configuring logging here it would be better to get the logger from the current_app